.github/workflows/pr-build-live-branch.yml

```
@@ -0,0 +1,164 @@
+      - name: "Checkout repository"
+        uses: actions/checkout@v4
+
+      - name: "Get current version"
+        id: current-version
+        run: |
+          VERSION=$( grep -oP '(?<=Version: )(.+)' woocommerce-gateway-stripe.php | head -n1 )
+          echo "Current version found: $VERSION" >> $GITHUB_STEP_SUMMARY
+          echo "VERSION=$VERSION" >> $GITHUB_OUTPUT
+
+      - name: "Set up PHP and Node"
+        uses: ./.github/actions/setup-env
```

actionlint (1.7.12)

[error] could not parse action metadata in "/home/jailuser/git/.github/actions/setup-env": line 5: unexpected key "type" for definition of input "php-version"

(action)